initdb/01.sql

```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE `bumblebee`;
+CREATE DATABASE `keycloak`;
```

Comment:
For safety reasons, you could add a check to avoid modifications if the table already exists:
```suggestion
CREATE DATABASE IF NOT EXISTS `keycloak`;
```